tests/testthat/test-FilterStates.R

```diff
@@ -43,10 +43,10 @@ testthat::test_that("set_filter_state sets count_type", {
 
 testthat::test_that("set_filter_state ignores teal_slice for inexisting variables with log warning", {
   filter_states <- FilterStates$new(data = data.frame(a = 1), dataname = "test")
-  res <- utils::capture.output(
-    filter_states$set_filter_state(teal_slices(teal_slice(dataname = "test", varname = "inexisting")))
+  testthat::expect_warning(
+    filter_states$set_filter_state(teal_slices(teal_slice(dataname = "test", varname = "inexisting"))),
+    "inexisting excluded from test"
```

Comment:
Please replace "inexisting" with "non-existent" in the four instances throughout the package.
```suggestion
    "non-existent excluded from test"
```